django/db/migrations/autodetector.py

```diff
@@ -820,7 +820,11 @@ def _generate_added_field(self, app_label, model_name, field_name):
         preserve_default = True
         if (not field.null and not field.has_default() and
                 not isinstance(field, models.ManyToManyField) and
-                not (field.blank and field.empty_strings_allowed)):
+                not (field.blank and field.empty_strings_allowed) and
+                not (
+                    isinstance(field, (models.DateField, models.DateTimeField, models.TimeField)) and
```

Comment:
No, it's unrelated to the underlying database column type, but part of these particular field implementations to define what the "default" value is.